internal/controller/constants.go

```diff
@@ -0,0 +1,35 @@
+package controller
+
+const (
+	/*** application server configuration file ***/
+	// OLSConfigName is the name of the OLSConfig configmap
+	OLSConfigCmName = "olsconfig"
+	// OLSNamespaceDefault is the default namespace for OLS
+	OLSNamespaceDefault = "openshift-lightspeed-operator"
+	// OLSAppServerServiceAccountName is the name of service account running the application server
+	OLSAppServerServiceAccountName = "lightspeed-app-server"
+	// OLSAppServerDeploymentName is the name of the OLS application server deployment
+	OLSAppServerDeploymentName = "lightspeed-app-server"
+	// APIKeyMountRoot is the directory hosting the API key file in the container
+	APIKeyMountRoot = "/etc/apikeys"
+	// BamApiKeyFileName is the name of the file containing the BAM API key in the secret referenced by the OLSConfig
+	BamApiKeyFileName = "bam_api_key.txt"
+	// OpenaiApiKeyFileName is the name of the file containing the OpenAI API key in the secret referenced by the OLSConfig
+	OpenaiApiKeyFileName = "openai_api_key.txt"
+	// OLSConfigFilename is the name of the application server configuration file
+	OLSConfigFilename = "olsconfig.yaml"
+	// Image of the OLS application server
+	// todo: image vesion should synchronize with the release version of the lightspeed-service-api image.
+	OLSAppServerImage = "quay.io/openshift/lightspeed-service-api:latest"
```

Comment:
this should be in the operator's cluster service version yaml rather than a constant in the code

Reply:
I add an arugment to the operator for passing image references for components. In the deployment we will pass the lightspeed-service-api image in the argument `-images=lightspeed-service=quay.io/openshift/lightspeed-service-api:latest`. 
I suggest we create the cluster service version yaml file in another [task to create the bundle](https://issues.redhat.com/browse/OLS-148).